### PR TITLE
Rename experiment to fix 16 char requirement.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
Tagging @dotnet/roslyn-ide

**Comment by CyrusNajmabadi** [comment]
Telling @MattGertz about this.  The AB infrastructure requires experiment names be 16 characters or less.  This is just adjusting the name we initially picked for OOP AB testing to meet that requirement.

**Comment by CyrusNajmabadi** [comment]
Retest windows_release_vs-integration_prtest please

**Comment by CyrusNajmabadi** [comment]
Telling @MattGertz about this.  This just fixes a tiny issue that we need if we want to have an AB experiement to enable to running things in OOP.

File: src/Workspaces/Core/Portable/Experiments/IExperimentationService.cs [hunk 1]
```diff
@@ -19,6 +19,6 @@ internal class DefaultExperimentationService : IExperimentationService
 
     internal static class WellKnownExperimentNames
     {
-        public const string OutOfProcessAllowed = nameof(OutOfProcessAllowed);
+        public const string RoslynFeatureOOP = nameof(RoslynFeatureOOP);
```
Comment (jasonmalinowski) [inline src/Workspaces/Core/Portable/Experiments/IExperimentationService.cs]: @CyrusNajmabadi Should this have a comment stating the limitation?

**Pull request merged**